extensions/amazon-lambda/maven-archetype/src/main/resources/archetype-resources/build.gradle

```diff
@@ -0,0 +1,32 @@
+plugins {
+    id 'java'
+    id 'io.quarkus'
+}
+
+repositories {
+     mavenLocal()
+     mavenCentral()
+}
+
+dependencies {
+    implementation enforcedPlatform("${quarkusPlatformGroupId}:${quarkusPlatformArtifactId}:${quarkusPlatformVersion}")
+    implementation 'io.quarkus:quarkus-resteasy'
+    implementation 'io.quarkus:quarkus-amazon-lambda'
+
+    testImplementation  "io.quarkus:quarkus-test-amazon-lambda"
+    testImplementation 'io.quarkus:quarkus-junit5'
+    testImplementation 'io.rest-assured:rest-assured'
+}
+
```

Comment:
When shipping the native binary to Lambda we need to create the `function.zip` file with the binary renamed to `bootstrap` insight. Can we use the a combination of the `distribution` plugin, `distZip` or custom `buildZip` task to do that?

Reply:
Don't worry too much about that.  I'm currently working on having the extension generate the zip.

Reply:
Quarkus will generate this for you, via gradle buildNative.  No need for custom tasks.

Reply:
@patriot1burke okay, so for now I rename the runner to bootstrap and zip it up to `function.zip`?
@oztimpower , quarkus will generate the runner but atm only mvn is creating the `function.zip`. As I understand this will be covered by @patriot1burke 's extension.

Reply:
Yes apologies, I forgot I had similar workarounds as you on the function.zip with Gradle, I did it via a one liner in manage.sh, and passed a flag if it was gradle or maven (to toggle the target/build).  Maybe I should PR that also.  For me I've always kept a Maven and Gradle build up to date on the projects as Gradle is only Preview and the Maven version put simply "just works"